e2e/tests/helpers.bash

```diff
@@ -341,4 +341,5 @@ cleanup() {
   delete_entities secret
   delete_entities eventdriver
   delete_entities eventdrivertype
+  sleep 5s
```

Comment:
Karol pushed a pr which fixed the race condition issue of our e2e. Is this trying to solve the same problem?

Reply:
Yeah, this can be removed at this point.

Reply:
Yes